specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-07-01-preview/workspaceRP.json

```diff
@@ -5682,6 +5682,9 @@
         }
       ],
       "properties": {
+        "appGwTargetFQDN": {
```

Comment:
1. We shouldn't use non common used abbreviations in property names, like Gw
2. This may be not only for appGw.
3. The word "Target" is confusing here. It's the frontend IPConfig FQDN. 
4. There could be multiple FQDNs associated with one private endpoint. 

my suggestion: string[] fqdns  (all lower case)

Reply:
done